flink-connectors/flink-connector-rabbitmq/src/main/java/org/apache/flink/streaming/connectors/rabbitmq/common/RMQConnectionConfig.java

```diff
@@ -506,16 +532,45 @@ public Builder setPrefetchCount(int prefetchCount) {
             return this;
         }
 
+        /**
+         * Enables setting the next message delivery timeout in the queueing consumer. Only
+         * applicable to the {@link RMQSource}. Set to 0 for unlimited (the consumer will be blocked
+         * until an element becomes available). Default is 30000.
```

Comment:
Nit: If you move the constant variable out of this class please also update the comments.